lib/taskana-core/src/main/java/pro/taskana/spi/task/internal/BeforeRequestChangesManager.java

```diff
@@ -0,0 +1,50 @@
+package pro.taskana.spi.task.internal;
+
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.api.exceptions.SystemException;
+import pro.taskana.common.internal.util.SpiLoader;
+import pro.taskana.spi.task.api.BeforeRequestChangesProvider;
+import pro.taskana.task.api.models.Task;
+
+public class BeforeRequestChangesManager {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AfterRequestChangesManager.class);
+
+  private final List<BeforeRequestChangesProvider> beforeRequestChangesProviders;
+
+  public BeforeRequestChangesManager(TaskanaEngine taskanaEngine) {
+    beforeRequestChangesProviders = SpiLoader.load(BeforeRequestChangesProvider.class);
+    for (BeforeRequestChangesProvider serviceProvider : beforeRequestChangesProviders) {
+      serviceProvider.initialize(taskanaEngine);
+      LOGGER.info(
+          "Registered BeforeRequestChanges service provider: {}",
```

Comment:
Can we agree on one of the wordings, "BeforeRequestChanges service provider" or "BeforeRequestChangesProvider service provider"?

Reply:
done